### PR TITLE
Merge Riak TS changes into develop

### PR DESCRIPTION
**Pull request by lukebakken**

### CODE REVIEW

**Comment by lukebakken** [comment]
I'm in the process of merging the timeseries changes for Riak Erlang Client into the master branch (basho/riak-erlang-client#294). This requires that a new version of `riak_pb` be available because the `2.1.3.0` tag does not compile using OTP 18.3. I am assuming this merge will be tagged as 2.1.4.0

Please comment on caveats I may not have considered.

@alexmoore @hmmr @macintux @lehoff @javajolt @erikleitch @andytill @gordonguthrie

**Comment by javajolt** [comment]
@lukebakken Sorry about that. I meant to tag it `2.1.4.0` last week, but got distracted with my trip. Yes, a new tag would be wonderful.

**Comment by javajolt** [comment]
Also I need to update lots of dependencies when that happens, so let me know.  Thanks!

**Comment by lukebakken** [comment]
@javajolt to confirm, this tag will be `develop` + `riak_ts-develop`. I'll run tests, erlang client tests, etc etc etc

**Comment by javajolt** [comment]
@lukebakken Yes. I thought we were already had switched to the TS one anyway. Only concern would be talking to old Riaks, but I don't think from a PB-perspective this would be a problem.

File: rebar.config [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 {deps, [
     {protobuffs, "0.8.*", {git, "git://github.com/basho/erlang_protobuffs.git", {tag, "0.8.2"}}},
-    {hamcrest, ".*", {git, "https://github.com/hyperthunk/hamcrest-erlang.git", {branch, "master"}}}
+    {hamcrest, ".*", {git, "https://github.com/basho/hamcrest-erlang.git", {tag, "0.2.0-basho"}}}
```
Comment (andytill) [inline rebar.config]: Is this used?

Comment (lukebakken) [inline rebar.config]: Sure doesn't appear that way:

```
lbakken@brahms ~/Projects/basho/riak_pb (features/lrb/merge-develop-and-riak_ts-develop $=)
$ git grep hamcrest
rebar.config:10:    {hamcrest, ".*", {git, "https://github.com/basho/hamcrest-erlang.git", {tag, "0.2.0-basho"}}}
```

Comment (javajolt) [inline rebar.config]: It isn't? So glad I had Paul fork it last release (facepalm)

Comment (lukebakken) [inline rebar.config]: Well actually:

```
==> riak_pb (dialyzer)
Unknown types:
  hamcrest:matchspec/0
make: *** [tools.mk:39: dialyzer-run] Error 1
```

Seems like a really lame reason to pull it in, but there it is.

**Pull request merged**